msal/src/main/java/com/microsoft/identity/client/internal/controllers/BrokerMsalController.java

```diff
@@ -246,6 +246,7 @@ public AcquireTokenResult acquireToken(InteractiveTokenCommandParameters paramet
     // The order matters.
     private List<BrokerBaseStrategy> getStrategies() {
         final List<BrokerBaseStrategy> strategies = new ArrayList<>();
+        strategies.add(new BrokerContentProviderStrategy());
```

Comment:
Should we try authService first for a couple of release? Once it's tried and tested (i.e. with Teams), then we can switch it to the first one in the list.

Reply:
Teams needs to test this on prod brokers right, if we add `AuthService` first, they will still run into this issue and `ContentProvider` fallback might not work. `ContentProvider` does the data transfer using android shared memory but it still uses `Binder` to communicate, and if the process is dead due to the exception before, this can' communicate as well.

Also, Since we always fallback to other strategies, it should work like before